docs/metrics-events.md

```diff
@@ -26,14 +26,16 @@ metrics queries in redash:
   * [Success event names](#success-event-names)
     * [View names](#view-names)
     * [Action names](#action-names)
-    * [Connect method names](#connect-method-names)
```

Comment:
Moved this one down a bit because links and experiments seem more generally interesting / higher precedence than CAD methods.